pkg/jobs/scheduled_job_executor.go

```diff
@@ -54,8 +55,14 @@ type ScheduledJobExecutor interface {
 	// GetCreateScheduleStatement returns a `CREATE SCHEDULE` statement that is
 	// functionally equivalent to the statement that led to the creation of
 	// the passed in `schedule`.
-	GetCreateScheduleStatement(ctx context.Context, env scheduledjobs.JobSchedulerEnv,
-		txn *kv.Txn, sj *ScheduledJob, ex sqlutil.InternalExecutor) (string, error)
+	GetCreateScheduleStatement(
+		ctx context.Context,
+		env scheduledjobs.JobSchedulerEnv,
+		txn *kv.Txn,
+		descsCol *descs.Collection,
+		sj *ScheduledJob,
+		ex sqlutil.InternalExecutor,
+	) (string, error)
```

Comment:
I wonder if it would have been better to add a method on JobSchedulerEnv instead (e.g. some sort of resolver type method).  The env was meant to be an environment for accessing things outside the schedule itself -- which is what it is...  Not a super strong preference, but please consider it.